src/libserver/http/http_router.c

```diff
@@ -40,6 +41,7 @@ static const struct _rspamd_http_magic {
 		[HTTP_MAGIC_HTML] = { "html", "text/html" },
 		[HTTP_MAGIC_CSS] = { "css", "text/css" },
 		[HTTP_MAGIC_JS] = { "js", "application/javascript" },
+		[HTTP_MAGIC_PNG] = { "png", "image/x-icon" },
```

Comment:
Is there a copy-paste error here? Shouldn't this line read:

`[HTTP_MAGIC_ICO] = { "ico", "image/x-icon" },`

instead?